.github/workflows/checks.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       group: lint-${{ github.ref }}
       cancel-in-progress: true
     container:
-      image: paritytech/ci-linux:production
+      image: paritytech/ci-unified:latest
```

Comment:
How about using a concrete version? Like `paritytech/ci-unified:bullseye-1.71.0-2023-05-23-v20230721`
This should prevent unexpected CI failures in case of breaking back-compatibility

Reply:
I like that

Reply:
@vmarkushin I agree. Actually I also think that using static tags is better than dynamic like: `:latest`